dq-game/src/utils/renderer.py

```diff
@@ -254,6 +254,10 @@ def render_timer(seconds):
 
         self.timer.reset(seconds, render_timer, timeout_callback)
 
+    @flush
+    def stop_timer(self):
+        self.timer.stop()
```

Comment:
If you created a function to stop the timer, use it everywhere. 
server_renderer.py line 64

Reply:
Not solved